<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlicingAlgorithmDialog</class>
 <widget class="QDialog" name="SlicingAlgorithmDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="input_layout">
     <item>
      <widget class="QLabel" name="lbl_workspace_input">
       <property name="text">
        <string>Input Workspace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MantidQt::MantidWidgets::WorkspaceSelector" name="workspace_selector">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="extents_layout">
     <item>
      <widget class="QPushButton" name="btn_calculate">
       <property name="toolTip">
        <string>Calculate axis-aligned extents from the workspace dimension limits</string>
       </property>
       <property name="text">
        <string>Calculate Extents</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ck_calculate">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Always calculate axis-aligned extents from the workspace dimension limits</string>
       </property>
       <property name="text">
        <string>Always Calculate Extents</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="axis_aligned_layout">
     <property name="title">
      <string> Axis-Aligned Binning</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QCheckBox" name="ck_axis_aligned">
          <property name="text">
           <string>Axis Aligned</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="non_axis_aligned_layout">
     <property name="title">
      <string>Non Axis-Aligned Binning</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="non_axis_aligned_core_layout">
        <item>
         <layout class="QHBoxLayout" name="output_bins_layout">
          <item>
           <widget class="QLabel" name="lbl_output_bins">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Output Bins</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_output_bins"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="output_extents_layout">
          <item>
           <widget class="QLabel" name="lbl_output_extents">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Output Extents</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_output_extents"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="translation_layout">
          <item>
           <widget class="QLabel" name="lbl_translation">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Translation</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_translation"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="ck_force_orthogonal">
          <property name="text">
           <string>Force Orthogonal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ck_normalisebasisvectors">
          <property name="text">
           <string>Normalise Basis Vectors</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="file_backend_layout">
     <property name="title">
      <string>File Back-End</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="output_layout_2">
        <item>
         <widget class="QLabel" name="lbl_output_filename">
          <property name="text">
           <string>Output Filename</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_filename"/>
        </item>
        <item>
         <widget class="QPushButton" name="btn_browse">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="memory_layout">
        <item>
         <widget class="QLabel" name="lbl_memory">
          <property name="text">
           <string>Memory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_memory"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="output_layout">
     <item>
      <widget class="QLabel" name="lbl_workspace_output">
       <property name="text">
        <string>Output Workspace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_output"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="recursion_layouts">
     <item>
      <widget class="QCheckBox" name="ck_max_from_input">
       <property name="text">
        <string>Take Max Recursion Depth From Input</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="recursion_layout">
       <item>
        <widget class="QLabel" name="lbl_resursion_depth">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Max Recursion Depth</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_resursion_depth">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="ck_parallel">
     <property name="text">
      <string>Parallel</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_help">
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="controls">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::WorkspaceSelector</class>
   <extends>QComboBox</extends>
   <header>MantidQtMantidWidgets/WorkspaceSelector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>controls</sender>
   <signal>accepted()</signal>
   <receiver>SlicingAlgorithmDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>controls</sender>
   <signal>rejected()</signal>
   <receiver>SlicingAlgorithmDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
